cmd/serve.go

```diff
@@ -19,6 +19,7 @@ type ServeCmd struct {
 	CleaningPeriod                           time.Duration `default:"600s" help:"prometheus metrics cleaning interval (for vanished nodes)"`
 	ElasticsearchConsulTag                   string        `default:"maintenance-elasticsearch" help:"elasticsearch consul tag"`
 	ElasticsearchEndpointSuffix              string        `default:".service.{dc}.foo.bar" help:"Suffix to add after the consul service name to create a valid domain name"`
+	ElasticsearchEndpointPort                int           `default:"-1" help:"Elasticsearch port used for cluster authentication"`
```

Comment:
Default should be 0 to be consistent with how Golang handle defaults (int defaults to 0).

Reply:
Done